batavia/modules/base64.js

```diff
@@ -0,0 +1,64 @@
+batavia.modules.base64 = {
+	__doc__: "",
+	__name__: "base64",
+	__file__: "base64.js",
+	__package__: "",
+
+	b64encode: function(data, altchars){
+		altchars = altchars || 0;
+		if (altchars === 0){
+			return new batavia.types.Str(window.btoa(data));
+		}
+	},
+
+	b64decode: function(data){
+		if (data.len % 4 !== 0) {
+			throw new batavia.builtins.ValueError("Incorrect padding");
+		}
+		return new batavia.types.Str(window.atob(data));
+	},
+
+	_a85chars: function(){},
+	_a85chars2: function(){},
+	_b32alphabet: function(){},
+	_b32rev: function(){},
+	_b32tab2: function(){},
+	_b85alphabet: function(){},
+	_b85chars: function(){},
+	_b85chars2: function(){},
+	_b85dec: function(){},
+	_bytes_from_decode_data: function(){},
+	_input_type_check: function(){},
+	_urlsafe_decode_translation: function(){},
+	_urlsafe_encode_translation: function(){},
+	a85decode: function(){},
+	a85encode: function(){},
+	b16decode: function(){},
+	b16encode: function(){},
+	b32decode: function(){},
+	b32encode: function(){},
+	b85decode: function(){},
+	b85encode: function(){},
+	binascii: function(){},
+	bytes_types: function(){},
+	decode: function(){},
+	decodebytes: function(){},
+	decodestring: function(){},
+	encode: function(){},
+	encodebytes: function(){},
+	encodestring: function(){},
+	main: function(){},
+	re: function(){},
+	standard_b64decode: function(){},
+	standard_b64encode: function(){},
+	struct: function(){},
+	test: function(){},
+	urlsafe_b64decode: function(){},
+	urlsafe_b64encode: function(){},
+
+};
+
+batavia.modules.base64.isclose.__python__ = true;
```

Comment:
Yes, it was, but now I'm having some trouble with conversion, since the functions should return bytes, and the atob and btoa methods return strings I have to convert them, and this is throwing some mistakes regarding the type